### DISCUSSION TITLE
(2025 update: not anymore – we’re back! 🎉) If this repository is dead, it should be marked as archived

### DISCUSSION THREAD

**Discussion by hnykda** [EPMatt/awesome-ha-blueprints#549] [Q&A] [answered]
Hi.
It seems to me @EPMatt , that the repository is not maintained anymore (last commit was more than a half year ago, and the PRs are piling up without getting merged). I am grateful for your contributions, but if you don't plan to work on this going forward, I propose that you mark is as archived and possibly hand the ownership to someone else. I am quite confident that it would do a good service to the blueprints ecosystem :-) .

**Comment by KrzysztofHajdamowicz** [comment]
Looks like @lsismeiro https://github.com/lsismeiro/awesome-ha-blueprints took over maintaining this repository

**Reply by victorhooi** [reply]
Right - that seems to be a fork.

However, it would still be good if *this* repo was marked as Archived?

Who is able to do that?

**Reply by hnykda** [reply]
The author...

**Comment by EPMatt** [comment, marked as answer]
Hey @hnykda, @KrzysztofHajdamowicz @victorhooi  and everyone who contributed to the discussion here,

great news - after a long hiatus, [we are back](https://github.com/EPMatt/awesome-ha-blueprints/discussions/746)! 🚀

I know it must have been frustrating to see this project slow down, and I can’t thank you all enough for keeping it alive while I was away. Looking at the time, effort, and dedication you've put in truly leaves me speechless. This is exactly what motivates me to jump back in and bring my energy back to the project. You all rock!

I’d love for you to check out the full [2025 update announcement](https://github.com/EPMatt/awesome-ha-blueprints/discussions/746) and share your thoughts there.

Thank you all for your amazing support and kind patience!  Let's make Awesome HA Blueprints even better, together. 💙
